src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import classNames from 'classnames';
+import { scaleLinear, ScaleLinear } from 'd3';
+
+import { getFeatureCoordinates } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+import { ProteinDomainsResources } from 'src/content/app/entity-viewer/types/product';
+
+import styles from './ProteinDomainImage.scss';
+
+const BLOCK_HEIGHT = 18;
+const BACKBONE_HEIGHT = 24;
```

Comment:
The metaphorical backbone is something that passes through the middle of a structure, like the stick passing through the series of rectangular blocks in transcript visualisation. Here, Andrea referred to these long things that contain rectangles as _tracks_.

Reply:
I've changed BACKBONE to TRACK.